app/assets/stylesheets/application.css.scss

```diff
@@ -15,8 +15,29 @@
  *= require foundation_and_overrides
  */
 
+.logo {
+  background-color: #d9b82d;
+  border-color: #111111;
+  margin-bottom: 0px;
+  margin-top: 10%;
+  text-align: center;
+  text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
+}
+
+.logo h1 {
+  border-radius: 0.1em;
+  color: #111111;
+  font-family: Impact;
+  font-size: 8em;
+}
+
+.twitter {
+  text-align: center;
+  border: none;
+}
+
 .background-image {
-  background: asset_url('admission.jpg') no-repeat center center fixed;
+  background: asset_url('theater.jpg') no-repeat center center fixed;
```

Comment:
Name of function `asset_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings